src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataFetcher.java

```diff
@@ -23,6 +23,7 @@
     private final Optional<URL> bloggersUrlOptional;
     private final Optional<URL> companiesUrlOptional;
     private final BloggersDataUpdater bloggersDataUpdater;
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Because `refreshBloggersDataFor` is called multiple times, and there is no need to create multiple instances of `ObjectMapper` which is a heavy weight object.